api/client/service/update.go

```diff
@@ -157,6 +156,7 @@ func updateService(spec *swarm.ServiceSpec, flags *pflag.FlagSet) error {
 		updateDurationOpt((flagRestartWindow), task.RestartPolicy.Window)
 	}
 
+	// TODO: needs to be a merge
```

Comment:
Let's handle this in this PR or in a followup before 1.12. We don't want to change the behavior later.